### PR TITLE
Add CRNN Chinese Support

### PR DESCRIPTION
**Pull request by zhtmike**
Continue work on #280 .

Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [ ] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: docs/cn/tutorials/training_recognition_custom_dataset_CN.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ word_1814.png	cathay
 ## 字典准备
 
 为训练中、英文等不同语种的识别网络，用户需配置对应的字典。只有存在于字典中的字符会被模型正确预测。MindOCR现提供中、英两种字典，其中
```
Comment (SamitHuang) [inline docs/cn/tutorials/training_recognition_custom_dataset_CN.md:54]: 请把默认英文字典 “012..abcd..”的说明加上。

Comment (zhtmike) [reply docs/cn/tutorials/training_recognition_custom_dataset_CN.md]: 已更新、

**Review by SamitHuang — APPROVED** [review]

File: configs/rec/crnn/README_CN.md [hunk 2]
```diff
@@ -354,6 +355,28 @@ Mindocr内置了一部分字典，均放在了 `mindocr/utils/dict/` 位置，
 - 请记住检查配置文件中的 `dataset->transform_pipeline->RecCTCLabelEncode->lower` 参数的值。如果词典中有大小写字母而且想区分大小写的话，请将其设置为 False。
 
 
+## 5. 多语言训练
```
Comment (HaoyangLee) [inline configs/rec/crnn/README_CN.md:358]: 英文README缺少这部分内容？

File: configs/rec/crnn/README_CN.md
Comment (HaoyangLee) [inline configs/rec/crnn/README_CN.md:1]: 中文README引用参考文献的跳转链接失效。修改："#references" -> "#参考文献"

**Review by HaoyangLee — APPROVED** [review]

File: configs/rec/crnn/README_CN.md (hunk 2 above)
Comment (zhtmike) [reply configs/rec/crnn/README_CN.md:358]: 有的 ##5. Multi-language Training

**Pull request merged**